ui/src/pages/profile.js

```diff
@@ -132,10 +132,12 @@ const Profile = ({ location }) => {
       endTime: moment().unix(),
       projects:
         data && data.user
-          ? data.user.projects.reduce(
-              (acc, current) => acc.concat(current.id),
-              [],
-            )
+          ? data.user.projects.reduce((acc, current) => {
+            if(current) {
```

Comment:
we could keep it simple `return current && current.Id ? acc.concat(current.id) : acc;`